_editing/4_editable_regions.md

```diff
@@ -46,3 +46,13 @@ Gives options for bold, italics and links
 On block elements you can set headings, bold, italics, links, quotes, lists and images.
 
 ![Block Elements](/img/editing/7.png)
+
+### Note about editable class in _includes
+
+If you add a class "editable" to an element in a file in _includes directory and then include it with 
+{% highlight liquid %}
+{% raw %}
+{% include your_file.html %}
+{% endraw %}
+{% endhighlight %}
+You will not be able to edit it in the Visual Editor (in Cloudcannon).
```

Comment:
There's no need to add the editable class to elements in includes as it won't do anything until we support it. If you using front matter to update an image you don't need the class of editable.